rdwatch/scoring/tasks/animation_export.py

```diff
@@ -0,0 +1,698 @@
+    prefix = f'{base_str}\
+        _{str(site_evaluation.site_id)}'
+    if frames:
+        # Create a temporary directory
+        self.update_state(
+            state='Progress',
+            meta={
+                'current': count,
+                'total': total_images,
+                'mode': f'Generating {output_format.upper()}',
+                'siteEvalId': site_evaluation_id,
+            },
+        )
+
+        output_dir = '/tmp/animation'
+        if not os.path.exists(output_dir):
+            os.mkdir(output_dir)
+        if output_format == 'gif':
+            # Create a temporary file for GIF
+            temp_file = tempfile.NamedTemporaryFile(
+                prefix=prefix, suffix='.gif', dir=output_dir, delete=False
+            )
+            with open(temp_file.name, 'w') as tmp_file:
+                output_file_path = tmp_file.name
+                frames[0].save(
+                    output_file_path,
+                    save_all=True,
+                    append_images=frames[1:],
+                    duration=1000.0 / fps,
+                    loop=0,
+                    optimize=False,
+                    quality=100,
+                )
+        else:  # 'mp4'
+            # Create a temporary file for MP4
+            temp_file = tempfile.NamedTemporaryFile(
+                prefix=prefix, suffix='.mp4', dir=output_dir, delete=False
+            )
+            with open(temp_file.name, 'w') as tmp_file:
+                output_file_path = tmp_file.name
+
+                # Initialize video writer
+                fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+                video_width = max_width_px
+                video_height = max_height_px
+                if rescale:
+                    video_width = rescaled_max_width
+                    video_height = rescaled_max_height
+                video_writer = cv2.VideoWriter(
+                    output_file_path, fourcc, fps, (video_width, video_height)
+                )
+
+                # Write each frame to the video
+                for frame in np_array:
+                    frame_bgr = cv2.cvtColor(
+                        frame, cv2.COLOR_RGB2BGR
+                    )  # Convert RGB to BGR for OpenCV
+                    video_writer.write(frame_bgr)
+
+                video_writer.release()  # Close the video writer
+        name = f'{prefix}.{output_format}'
+        self.update_state(
+            state='SUCCESS',
+            meta={
+                'current': count,
+                'total': total_images,
+                'mode': 'Rendering Complete',
+                'siteEvalId': site_evaluation_id,
+            },
+        )
+
+        return output_file_path, name
+    return False, False
+
+
+@app.task(bind=True)
+def create_site_animation_export(
+    self, site_evaluation_id: UUID4, settings: GenerateAnimationSchema, userId: int
```

Comment:
Similar to above issue - 
```suggestion
    self, site_evaluation_id: UUID4, settings: dict[str, Any], userId: int
```